SmartDeviceLinkTests/RPCSpecs/RequestSpecs/SDLButtonPressSpec.m

```diff
@@ -54,11 +54,12 @@
     });
 
     it(@"Should get correctly using initializer", ^ {
-        SDLButtonPress *testRequest = [[SDLButtonPress alloc] initWithButtonName:SDLButtonNameAC moduleType:SDLModuleTypeClimate moduleId:@"123"];
```

Comment:
Deprecated initializers still need tests

Reply:
Added tests for deprecated initializers.